prometheus/metrics.go

```diff
@@ -144,8 +144,8 @@ func getNamespaceServicesRequestRates(ctx context.Context, api prom_v1.API, name
 // getServiceRequestRates retrieves traffic rates for requests entering, or internal to the namespace, for a specific service name
 // Note that it does not discriminate on "reporter", so rates can be inflated due to duplication, and therefore
 // should be used mainly for calculating ratios (e.g total rates / error rates)
-func getServiceRequestRates(ctx context.Context, api prom_v1.API, namespace, service string, queryTime time.Time, ratesInterval string) (model.Vector, error) {
-	lbl := fmt.Sprintf(`destination_service_name="%s",destination_service_namespace="%s"`, service, namespace)
+func getServiceRequestRates(ctx context.Context, api prom_v1.API, cluster, namespace, service string, queryTime time.Time, ratesInterval string) (model.Vector, error) {
+	lbl := fmt.Sprintf(`destination_service_name="%s",destination_service_namespace="%s",destination_cluster="%s"`, service, namespace, cluster)
```

Comment:
Do we want just traffic from the cluster that the service lives in, or from any cluster the traffic comes from?.

Reply:
I think you're right. We are interested in the traffic rates from any service/cluster to the destination service, which lives in a cluster.